### PR TITLE
integrate with level of achievements selector

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**
Needs:
- https://github.com/Brightspace/d2l-hm-constants-behavior/pull/72
- https://github.com/Brightspace/outcomes-level-of-achievement-ui/pull/8

### CODE REVIEW

**Comment by jstefaniuk-d2l** [comment]
@omsmith @awikkerink (I can't add reviewers)

File: d2l-alignment.html [hunk 1]
```diff
@@ -37,10 +37,23 @@
 				margin-right: 0;
 				margin-left: 2px;
 			}
+
+			d2l-outcomes-level-of-achievements {
+				display: block;
```
Comment (omsmith) [inline d2l-alignment.html]: Consider setting this in `:host{}` for the `d2l-outcomes-level-of-achievements` itself?

File: d2l-outcome.html [hunk 2]
```diff
@@ -14,6 +14,8 @@
 			:host {
 				display: block;
 				width: 100%;
+				@apply --d2l-body-small-text;
```
Comment (omsmith) [inline d2l-outcome.html]: Do we need _d2l-typography_ for this to work? It's currently a dev depenency

File: d2l-alignment.html [hunk 3]
```diff
@@ -5,19 +5,19 @@
 <link rel="import" href="../d2l-polymer-siren-behaviors/store/entity-behavior.html">
 <link rel="import" href="../d2l-polymer-siren-behaviors/store/siren-action-behavior.html">
 <link rel="import" href="../d2l-hm-constants-behavior/d2l-hm-constants-behavior.html">
+<link rel="import" href="../d2l-outcomes-level-of-achievements-ui/d2l-outcomes-level-of-achievements.html">
```
[the remark this answers is not in the document]
Comment (jstefaniuk-d2l) [reply d2l-alignment.html]: - [ ] Add this to bower once it's PR gets merged

**Comment by jstefaniuk-d2l** [comment]
@awikkerink @omsmith  Any last feedback? 😄

**Comment by omsmith** [comment]
@awikkerink you happy?

**Pull request merged**